src/Dofs/ConstraintHandler.jl

```diff
@@ -745,11 +745,12 @@ function _condense!(K::SparseMatrixCSC, f::AbstractVector, dofcoefficients::Vect
         col_coeffs = coefficients_for_dof(dofmapping, dofcoefficients, col)
         if col_coeffs === nothing
             for a in nzrange(K, col)
+                Kval = K.nzval[a]
+                iszero(Kval) && continue
```

Comment:
Kval will most likely not be zero, right? So maybe it is an unnecessary check?

Reply:
I don't think it is that uncommon? I.e. if you have no coupling between some dofs?

Reply:
From the heat_equation tutorial, the stiffness matrix has no zero values (before we call apply!) i.e. `count(i-> i==0, K.nzval) == 0` . But idk, maybe it has many zeros it in other cases?

Reply:
That has only one field. But if you have multiple fields it's often that some fields don't couple in the weak form.